swarm/tools/flow_studio_ui/index.html

```
@@ -49,7 +49,7 @@ <h1>Flow Studio</h1>
         <span id="governance-issues-count" class="governance-issues-badge" style="display: none;" aria-live="polite">0</span>
       </button>
       <label id="governance-toggle" class="governance-toggle author-only" title="Highlight governance issues on graph nodes" data-uiid="flow_studio.header.governance.overlay">
-        <input type="checkbox" id="governance-overlay-checkbox" />
+        <input type="checkbox" id="governance-overlay-checkbox" aria-label="Highlight governance issues on graph nodes" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)